2_adventures/001/ladanazita/04_multiply_character/multiply.rb

```diff
@@ -0,0 +1,3 @@
+def spam(number) 
+  'hue' *number 
```

Comment:
You really do want to have a space around your multiplication method name.  Otherwise it looks like a unary method to me and Ruby. but it is not a unary number.  Plus it is easier to read.  `1*1` compared to `1 * 1`.